src/components/QuickOpenModal.js

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 // @flow
+const path = require("path");
```

Comment:
will this work w/ webpack?

Reply:
I'm not sure how to test this

Reply:
I think you should be able to do `import path from "path";` and yeah Webpack should do the work here if I recall.